src/lib/expansion-panel/expansion-summary.scss

```diff
@@ -0,0 +1,60 @@
+$md-summary-height: 48px;
+$md-summary-height-expanded: 64px;
+
+.mat-summary {
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  height: $md-summary-height;
+  line-height: $md-summary-height;
+  padding: 0 24px;
+
+  .content {
+    display: flex;
+    flex: 1;
+    flex-direction: row;
+    overflow: hidden;
+  }
+
+  &.expanded {
+    height: $md-summary-height-expanded;
+    line-height: $md-summary-height-expanded;
+  }
+
+  &:focus, 
+  &:hover {
+    background: rgb(238, 238, 238);
```

Comment:
The color should be inside the theme. The same goes for `[title]`, `[description]` etc.